zh/notes-tidb-operator-v1.1.md

```diff
@@ -0,0 +1,104 @@
+---
+title: TiDB Operator v1.1 重要注意事项
+category: how-to
+---
+
+# TiDB Operator v1.1 重要注意事项
+
+本文介绍 TiDB Operator v1.1 版本重要注意事项。
+
+## PingCAP 不再继续更新维护 tidb-cluster chart
+
+从 TiDB Operator v1.1.0 开始，PingCAP 不再继续更新维护 tidb-cluster chart，原来由 tidb-cluster chart 负责管理的组件或者功能在 v1.1 中的变更如下：
+| 组件、功能 | v1.1 |
+| :--- | :--- |
+| TiDB Cluster (PD, TiDB, TiKV) | [TidbCluster CR](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) |
+| TiDB Monitor | [TidbMonitor CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/monitor/tidb-monitor.yaml) |
+| TiDB Initializer | [TidbInitializer CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/initializer/tidb-initializer.yaml) |
+| Scheduled Backup | [BackupSchedule CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-schedule-aws-s3-br.yaml) |
+| Pump | [TidbCluster CR](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) |
+| Drainer | [tidb-drainer chart](https://github.com/pingcap/tidb-operator/tree/master/charts/tidb-drainer) |
+| Importer | [tikv-importer chart](https://github.com/pingcap/tidb-operator/tree/master/charts/tikv-importer) |
+
+- tidb-cluster chart 仍然会继续发布，但是不再添加新功能，比如组件 TLS（目前只是部分支持）、TiKV 和 TiDB 自动伸缩等功能。
+- 通过 v1.0.x TiDB Operator 部署的 TiDB 集群，在 TiDB Operator 升级到 v1.1 之后，仍然可以通过 v1.1 版本的 tidb-cluster chart 升级和管理 TiDB 集群。
+
+## tidb-cluster chart 管理的组件或者功能切换到 v1.1 完整支持的方式
+
+虽然 TiDB Operator v1.1 版本仍然支持用户使用 Helm 和 tidb-cluster chart 管理集群，但是由于 tidb-cluster chart 不再新增功能，用户可以自己为 tidb-cluster chart 贡献新功能或者切换到 TiDB Operator v1.1 完整支持的方式。
+
+下面介绍如何将 tidb-cluster chart 管理的组件或者功能切换到 v1.1 完整支持的方式。
+
+### Discovery
+
+Discovery 服务直接由 TiDB Operator 内部生成，不再需要用户做任何配置。
+
+### PD、TiDB、TiKV
+
+在 tidb-cluster chart 中，PD、TiDB、TiKV 配置由 Helm 渲染成 ConfigMap，从 TiDB Operator v1.1 开始，PD、TiDB、TiKV 配置也可以直接在 TiDBCluster CR 中配置，具体配置方法可以参考 [TiDB 集群配置](configure-tidb-cluster.md)。
```

Comment:
I already suggest @Yisaer changing the file name in https://github.com/pingcap/docs-tidb-operator/pull/12#issuecomment-601532218
```suggestion
在 tidb-cluster chart 中，PD、TiDB、TiKV 配置由 Helm 渲染成 ConfigMap，从 TiDB Operator v1.1 开始，PD、TiDB、TiKV 配置也可以直接在 TiDBCluster CR 中配置，具体配置方法可以参考 [通过 TidbCluster 配置 TiDB 集群](configure-cluster-using-tidbcluster.md)。
```